bundles/org.openhab.binding.monopriceaudio/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,552 @@
+				<description>The Controls for Zone 3</description>
+			</channel-group>
+			<channel-group id="zone4" typeId="zone4">
+				<label>Zone 4</label>
+				<description>The Controls for Zone 4</description>
+			</channel-group>
+			<channel-group id="zone5" typeId="zone5">
+				<label>Zone 5</label>
+				<description>The Controls for Zone 5</description>
+			</channel-group>
+			<channel-group id="zone6" typeId="zone6">
+				<label>Zone 6</label>
+				<description>The Controls for Zone 6</description>
+			</channel-group>
+			<channel-group id="zone7" typeId="zone7">
+				<label>Zone 7</label>
+				<description>The Controls for Zone 7</description>
+			</channel-group>
+			<channel-group id="zone8" typeId="zone8">
+				<label>Zone 8</label>
+				<description>The Controls for Zone 8</description>
+			</channel-group>
+			<channel-group id="zone9" typeId="zone9">
+				<label>Zone 9</label>
+				<description>The Controls for Zone 9</description>
+			</channel-group>
+			<channel-group id="zone10" typeId="zone10">
+				<label>Zone 10</label>
+				<description>The Controls for Zone 10</description>
+			</channel-group>
+			<channel-group id="zone11" typeId="zone11">
+				<label>Zone 11</label>
+				<description>The Controls for Zone 11</description>
+			</channel-group>
+			<channel-group id="zone12" typeId="zone12">
+				<label>Zone 12</label>
+				<description>The Controls for Zone 12</description>
+			</channel-group>
+			<channel-group id="zone13" typeId="zone13">
+				<label>Zone 13</label>
+				<description>The Controls for Zone 13</description>
+			</channel-group>
+			<channel-group id="zone14" typeId="zone14">
+				<label>Zone 14</label>
+				<description>The Controls for Zone 14</description>
+			</channel-group>
+			<channel-group id="zone15" typeId="zone15">
+				<label>Zone 15</label>
+				<description>The Controls for Zone 15</description>
+			</channel-group>
+			<channel-group id="zone16" typeId="zone16">
+				<label>Zone 16</label>
+				<description>The Controls for Zone 16</description>
+			</channel-group>
+			<channel-group id="zone17" typeId="zone17">
+				<label>Zone 17</label>
+				<description>The Controls for Zone 17</description>
+			</channel-group>
+			<channel-group id="zone18" typeId="zone18">
+				<label>Zone 18</label>
+				<description>The Controls for Zone 18</description>
+			</channel-group>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="false">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>Serial Port to Use for Connecting to the Monoprice Amplifier</description>
+			</parameter>
+			<parameter name="host" type="text" required="false">
+				<context>network-address</context>
+				<label>Address</label>
+				<description>Host Name or IP Address of the Machine Connected to the Monoprice Amplifier (Serial over IP)</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false">
```

Comment:
You could specify min/max.

Reply:
fixed and changes pushed for another look...